### DISCUSSION TITLE
Javascript Core 支持情况?

### DISCUSSION THREAD

**Discussion by BlurryLight** [Tencent/puerts#1760] [Q&A] [answered]
https://github.com/puerts/backend-javascriptcore
有看到这个仓库最近在更新，但是puerts似乎还是没有把JavaScriptCore放到可以支持的backend。

想问一下在Unreal上这个目前还在开发阶段么，还是处于可用状态? 如果想要测试应该从哪里接入。

**Comment by chexiongsheng** [comment]
预研阶段。
你们对这感兴趣？
另外我一直想问下你哪个公司的。

**Reply by BlurryLight** [reply]
1. 想要验证一下，iOS上能有Jit更好。
2. 完美世界

**Reply by chexiongsheng** [reply, marked as answer]
那让你失望了，jscore在ios上也不能jit。
能不能jit在于os给不给权限（开辟可写可执行的内存），发布后ios下只有苹果自己的app，进程才有权限。普通应用用了jscore也无法jit。
如果给了权限，v8也能jit，比如xcode连接着调试，开发者账号授权（能维持7天还是多少天），v8也能jit。或者越狱也行。
